src/helpers/CBV/tests/ObjectList.test.tsx

```diff
@@ -0,0 +1,130 @@
+import reducerRegistry from '@onaio/redux-reducer-registry';
+import { mount } from 'enzyme';
+import React from 'react';
+import { Provider } from 'react-redux';
+import store from '../../../store';
+import reducer, {
+  Message,
+  reducerName,
+  removeMessagesAction,
+  selectAllMessages,
+  SEND_MESSAGE,
+  sendMessage,
+  SendMessageAction,
+} from '../../../store/tests/ducks/messages';
+import { ObjectList } from '../ObjectList';
+
+/** register reducers */
+reducerRegistry.register(reducerName, reducer);
+
+interface TestProps {
+  actionCreator?: typeof sendMessage;
+  callbackFunc?: () => void;
+  messages?: Message[];
+}
+
+const TestComponent = (_: TestProps) => <div>mosh</div>;
```

Comment:
Would it make sense to have this `TestComponent` be a list of elements driven by `props.messages` since it's a test for `ObjectList`?

Reply:
I'll do that for clarity in the test.  Thanks.